test/scenarios/ynEIGEN/ynLSDeDepositAdapter.t.sol

```diff
@@ -5,10 +5,14 @@ import {TestAssetUtils} from "test/utils/TestAssetUtils.sol";
 
 import "./ynLSDeWithdrawals.t.sol";
 
-contract ynLSDeDepositAdapterTest is ynLSDeWithdrawalsTest {
```

Comment:
this shouldn't inherit the ynLSDeWithdrawalsTest, as it reruns all its tests again.

Just the base